README.md

```diff
@@ -1,23 +1,39 @@
 # CU Reviews
 
-#### Contents
-- [About]
-- [Getting Started]
-- [Dependencies & Libraries]
-- [Screenshots]
-- [Contributors]
+## Contents
+
+- [CU Reviews](#cu-reviews)
+  - [Contents](#contents)
+  - [About](#about)
+  - [Getting Started](#getting-started)
+  - [Dependencies & Libraries](#dependencies--libraries)
+  - [Screenshots](#screenshots)
+  - [Contributors](#contributors)
+    - [2019-2020](#2019-2020)
+    - [2018-2019](#2018-2019)
+    - [2017-2018](#2017-2018)
 
 ## About
+
 A web app for Cornell students to read and write reviews for Cornell classes, allowing a common platform for students to get advice and suggestions for picking classes. In other words, a "rate my Cornell classes".
 
 ## Getting Started
-Make sure you have [Meteor](https://www.meteor.com) installed!
 
-    npm install
-    meteor npm install
-    meteor --port 3000
+Create .env into `server/` folder with content like:
+
+```bash
+MONGODB_URL=[SECRET_URL_ASK_TEAM_MEMBER_FOR_IT]
+```
+
+Make sure you have [Yarn](https://classic.yarnpkg.com/) installed.
 
-You can access the app at http://localhost:3000.
+```bash
+yarn
+yarn workspace server start # start server
+yarn workspace client start # start client
+```
+
+You can access the app at [http://localhost:3000].
 
 Next, add a password to your local database. Launch a second terminal window and
```

Comment:
I should mention that this is not how we do admin validation anymore. We set field on the user's mongodb entry.